about.html

```diff
@@ -0,0 +1,179 @@
+            <a href="#">fsowah001@gmail.com<i class="uil uil-angle-right"></i></a>
+        </div>
+    </section>
+    <section class="next-page" id="campaign">
+        <div class="container">
+            <h3>World Health Organization 1948 Logo</h3>
+            <div class="divide"></div>
+            <p>"The logo was decided through a comference held on October, 21 1948 in the
+                United States. If we are talking about resilience and equity,
+                we are talking about two sides of the same coin.
+                Those are both important objectives of health systems and they are heavily interlinked with each other.
+                "</p>
+            <div class="logo-about">
+                <img src="./images/logo.png" alt="logo">
+                <h3>WHO</h3>
+                <small>World Health Organization 1948</small>
+            </div>
+
+        </div>
+    </section>
+    <section class="main" id="program">
+        <div class="main-program">
+            <h2>See Past Global Health Summit</h2>
+            <div class="linear"></div>
+            <h4>Take a look at the last two WHO global summit which took place in Accra, Ghana</h4>
+        </div>
+        <div class="img-cont">
+            <div class="img-container">
+                <div class="summit-text">
+                    <h2>2015</h2>
+                    <p>WHO global summit in Accra, Ghana</p>
+                </div>
+            </div>
+            <div class="img-contain">
+                <div class="summit-content">
+                    <h2>2019</h2>
+                    <p>WHO global summit in Accra, Ghana</p>
+                </div>
+            </div>
+        </div>
+        
+        
+    </section>
+    <section class="partner2" id="sponsor">
+        <div class="partner-title2">
+            <h2>Our Partners</h2>
+            <div class="partner-divider2"></div>
+        </div>
+        <div class="partners2">
+            <ul>
+                <li>mozilla</li>
+                <li>Google</li>
+                <li>Amazon</li>
+                <li>daumkakao</li>
+            </ul>
+        </div>
+    </section>
+    <footer id="news" >
+        <div class="footer-logo">
+            <div class="logo2">
+                <a href="#"><img src="./images/logo.png" alt="logo"></a>
+                <a href="#">WHO</a>
+                <h5>World Health Organization</h5>
+            </div> 
+            <div class="rights">
+                <h5>1948 World Health Organization. Some Rights Reserved.</h5>
+                <p>The World Health Organization is a specialized agency of the United Nations 
+                    responsible for international public health. The WHO Constitution states its main objective as 
+                    "the attainment by all peoples of the highest possible level of health".</p>
+            </div>
+        </div>
+
+    </footer>

```

Comment:
The footer of the about page (desktop version) has a dark background whilst the section before it (See Past Global Health Summit) have a white background, but your design is the opposite. Kindly check and fix this. :pray: 
Also kindly check and make sure that all dimensions including width, margins, font-size, etc matches the design

### Your design

![Screenshot from 2023-01-04 12-59-23](https://user-images.githubusercontent.com/65233860/210561107-8d20b9d5-fd80-4c24-9ee8-18d3822bb5ed.png)




### The original design
 
![Screenshot from 2023-01-04 12-59-14](https://user-images.githubusercontent.com/65233860/210561137-6f5ed5e2-ea1f-4d96-be41-7b563853ebc3.png)